### PR TITLE
Clarified some language in the Globalization and Localization article

### PR DESCRIPTION
**Pull request by Rabadash8820**
I made this change to address user "Dazinator's" comment on the paragraph in question.  Hopefully this change will be clearer to future readers.

Suggested reviewers:

@Rick-Anderson
@damienbod
@hishamco

### CODE REVIEW

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -320,4 +320,4 @@ Terms:
 
 * [Localization.StarterWeb project](https://github.com/aspnet/entropy) used in the article.
 * [Resource Files in Visual Studio](https://docs.microsoft.com/cpp/windows/resource-files-visual-studio)
-* [Resources in .resx Files](https://docs.microsoft.com/dotnet/framework/resources/working-with-resx-files-programmatically)
\ No newline at end of file
+* [Resources in .resx Files](https://docs.microsoft.com/dotnet/framework/resources/working-with-resx-files-programmatically)
```
Comment (hishamco) [inline aspnetcore/fundamentals/localization.md]: What's changed here?!!

Comment (scottaddie) [reply aspnetcore/fundamentals/localization.md]: It's a difference in white space.

**Comment by scottaddie** [comment]
@Rabadash8820 Thank you for the contribution! This change will go live within the next 24 hours.

File: aspnetcore/fundamentals/localization.md (hunk 1 above)
Comment (hishamco) [reply aspnetcore/fundamentals/localization.md]: I didn't notice anything 😄

Comment (Rabadash8820) [reply aspnetcore/fundamentals/localization.md]: @hishamco  Yeah I wasn't sure what changed there either haha

**Pull request merged**